modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -540,6 +540,23 @@ class Editor implements EditorObservable {
    * @param {String/Function} cmdFunc Command name string or function to execute when the key is pressed.
    * @param {Object} scope Optional scope to execute the function in.
    * @return {Boolean} true/false state if the shortcut was added or not.
+   * @example
+   * editor.addShortcut('ctrl+a', "description of the shortcut", function() {});
+   * editor.addShortcut('ctrl+alt+a', "description of the shortcut", function() {});
+   * editor.addShortcut('meta+a', "description of the shortcut", function() {});
+   * // "meta" maps to Command on Mac and Ctrl on PC
+   * editor.addShortcut('access+a', "description of the shortcut", function() {});
+   * // "access" maps to Control+Option on Mac and shift+alt on PC
+   *
+   * editor.addShortcut(
+   *  'meta+access+c', 'Opens the code editor dialog.', function () {
```

Comment:
Why was this added on a new line? It's not too bad but it is a little odd, so would be better on a single line (likewise for the other examples).

Reply:
Unfortunately the CCS on the API docs is pretty messy. Overflowing content is hidden, not wrapped.